tembo-cli/src/cmd/apply.rs

```diff
@@ -142,6 +167,31 @@ pub fn get_instance_id(
     Ok(None)
 }
 
+pub fn is_instance_up(
+    instance_id: String,
+    config: &Configuration,
+    env: &Environment,
+) -> Result<Option<Box<ConnectionInfo>>> {
```

Comment:
has a `Result` return but never returns an error

Reply:
What would you recommend the best way for me to return the error here?